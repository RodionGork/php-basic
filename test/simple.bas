5 REM Simple Program 11!
10 x = 40
12 let y = x+5
15 PRINT "Meaning of the life is:", y-3
25 PRINT "Zlo is:"; : PRINT "naka"; "zuemo"
90 END
