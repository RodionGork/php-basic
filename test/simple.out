Meaning of the life is: 42
Zlo is:nakazuemo
